.github/workflows/release.yaml

```diff
@@ -106,7 +106,7 @@ jobs:
           # Binaries x86_64
           # TODO: add the rest of binaries
           # NOTE that we are not making the attestation fail if the material is not found. We will fail on "att push"
-          echo -n '${{ steps.release.outputs.artifacts }}' | jq -r '.[] | select(.type=="Binary" and .goos=="linux") | { "name": "\(.extra.ID)-\(.goos)-\(.goarch)", "path":"\(.path)"} | @base64' | while read i; do
+          echo -n '${{ steps.release.outputs.artifacts }}' | jq -r '.[] | select(.type=="Binary") | { "name": "\(.extra.ID)-\(.goos)-\(.goarch)", "path":"\(.path)"} | @base64' | while read i; do
```

Comment:
what if we start parsing the contract .github/workflows/contracts/chainloop-vault-release.yml instead ?